vita3k/main.cpp

```diff
@@ -364,14 +366,17 @@ int main(int argc, char *argv[]) {
 
             gui::draw_end(gui, emuenv.window.get());
             emuenv.renderer->swap_window(emuenv.window.get());
-            SDL_SetWindowTitle(emuenv.window.get(), fmt::format("{} | {} ({}) | Please wait, compiling shaders...", window_title, emuenv.current_app_title, emuenv.io.title_id).c_str());
         }
     }
 
     if (const auto err = run_app(emuenv, entry_point) != Success)
         return err;
 
+    SDL_SetWindowTitle(emuenv.window.get(), fmt::format("{} | {} ({}) | Please wait, loading...", window_title, emuenv.current_app_title, emuenv.io.title_id).c_str());
+
     while (emuenv.frame_count == 0 && !emuenv.load_exec) {
+        handle_events(emuenv, gui);
```

Comment:
is it possible to move this to the while loop? like here https://github.com/Vita3K/Vita3K/pull/2050/files#diff-3f32ac2d39aecf3b70002510995410c38a904e461b42e208565ba22cfbadd6c0R398. Or maybe the other way around, move the handle_events out of the while loop, just for consitency tho else, overall code is fine

Reply:
is possible yes